x-pack/legacy/plugins/epm/server/lib/elasticsearch/template/install.ts

```diff
@@ -26,6 +26,7 @@ export async function installTemplates(pkg: RegistryPackage, callCluster: CallES
   // If no datasets exist in this package, no templates have to be installed.
   if (!pkg.datasets) return;
   return pkg.datasets.map(async dataset => {
+    dataset.package = pkg.name;
```

Comment:
@ruflin what do you think about having the Registry supply this instead of EPM adding it?

Reply:
Yes: https://github.com/elastic/package-registry/pull/189